### PR TITLE
Fix/cat search fixes

### PR DESCRIPTION
**Pull request by AshTNA**

### CODE REVIEW

File: templates/search/blocks/search_results__selected-filters.html [hunk 1]
```diff
@@ -14,7 +14,7 @@ <h2 class="sr-only">Selected filters</h2>
 
     {% if selected_filters %}
         <li class="search-results__selected-filters-item-remove-all">
-            <a href="?q={{ form.cleaned_data.q }}&group={{ form.cleaned_data.group }}" class="search-results__selected-filters-item-remove-all-link">Remove all filters</a>
+            <a href="{% if form.cleaned_data.q %}?q={{ form.cleaned_data.q }}&{% else %}?{% endif %}group={{ form.cleaned_data.group }}" class="search-results__selected-filters-item-remove-all-link">Remove all filters</a>
```
Comment (ababic) [inline templates/search/blocks/search_results__selected-filters.html:17]: We should avoid complicating code here for the sake of 'url vanity'. It's fine for `q` to appear even if it is blank.

Comment (AshTNA) [reply templates/search/blocks/search_results__selected-filters.html:17]: Thanks @ababic - this is why I shouldn't touch the logic too much!

**Review by ababic — APPROVED** [review]
LGTM :)

**Comment by AshTNA** [comment]
Thanks @ababic !

**Pull request merged**